common/api/summary/ecschema-metadata.exports.csv

```diff
@@ -99,7 +99,7 @@ beta;PropertyCategory
 beta;PropertyCategoryProps 
 beta;PropertyProps
 beta;PropertyType
-beta;propertyTypeToString(type: PropertyType): "PrimitiveProperty" | "StructProperty" | "StructArrayProperty" | "NavigationProperty" | "PrimitiveArrayProperty"
+beta;propertyTypeToString(type: PropertyType): "PrimitiveArrayProperty" | "PrimitiveProperty" | "StructArrayProperty" | "StructProperty" | "NavigationProperty"
```

Comment:
I'd revert this file. It may trip up extract-api.

Run this command to fix, `git checkout master -- common/api/summary/ecschema-metadata.exports.csv`.

Reply:
Fixed!